nilearn/decoding/decoder.py

```diff
@@ -0,0 +1,818 @@
+        test data (X_test) and the test target (y_test) if y is
+        not None.
+        e.g. scorer(estimator, X_test, y_test)
+
+        For classification: 'accuracy', 'f1', 'precision', or 'recall'.
+            Default 'roc_auc'.
+
+    smoothing_fwhm : float, optional. Default: None
+        If smoothing_fwhm is not None, it gives the size in millimeters of the
+        spatial smoothing to apply to the signal.
+
+    standardize : bool, optional. Default: True
+        If standardize is True, the time-series are centered and normed:
+        their variance is put to 1 in the time dimension.
+
+    target_affine : 3x3 or 4x4 matrix, optional. Default: None
+        This parameter is passed to image.resample_img. Please see the
+        related documentation for details.
+
+    target_shape : 3-tuple of int, optional. Default: None
+        This parameter is passed to image.resample_img. Please see the
+        related documentation for details.
+
+    low_pass : None or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    high_pass : None or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    t_r : float, optional. Default: None
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details.
+
+    mask_strategy : {'background' or 'epi'}, optional. Default: 'background'
+        The strategy used to compute the mask: use 'background' if your
+        images present a clear homogeneous background, and 'epi' if they
+        are raw EPI images. Depending on this value, the mask will be
+        computed from masking.compute_background_mask or
+        masking.compute_epi_mask.
+
+    memory : instance of joblib.Memory or str
+        Used to cache the masking process.
+        By default, no caching is done. If a str is given, it is the
+        path to the caching directory.
+
+    memory_level : int, optional. Default: 0
+        Rough estimator of the amount of memory used by caching. Higher value
+        means more memory for caching.
+
+    n_jobs : int, optional. Default: 1.
+        The number of CPUs to use to do the computation. -1 means
+        'all CPUs'.
+
+    verbose : int, optional. Default: 0.
+        Verbosity level.
+
+    See Also
+    ------------
+    nilearn.decoding.Decoder - classification strategies for Neuroimaging.
+
+    """
+
+    def __init__(self, estimator='svc', mask=None, cv=10, param_grid=None,
+                 screening_percentile=20, scoring='roc_auc',
+                 smoothing_fwhm=None, standardize=True, target_affine=None,
+                 target_shape=None, mask_strategy='background',
+                 low_pass=None, high_pass=None, t_r=None, memory=None,
+                 memory_level=0, n_jobs=1, verbose=0):
+        super(Decoder, self).__init__(
```

Comment:
This is Python 2 style for calling super. In python 3 I think super().__init__(params) will do the job